tests/binning_test.py

```diff
@@ -507,3 +507,31 @@ def test_rebin_deprecated_positional_arguments():
     with pytest.warns(UserWarning):
         result = sc.rebin(da, 'x', x)
     assert sc.identical(result, da.rebin(x=x))
+
+
+@pytest.mark.parametrize('op', ['bin', 'hist', 'nanhist'])
+def test_raises_ValueError_given_variable_and_multiple_edges(op):
```

Comment:
It also raises when given no edges. Is this consistent with binning, etc. of data arrays?

Reply:
Depends. You can do `binned_data_array.hist()`, but `table.hist()` will raise.